apps/test-suite/tests/SQLite.ts

```diff
@@ -119,16 +117,16 @@ CREATE TABLE IF NOT EXISTS test (id INTEGER PRIMARY KEY NOT NULL, name VARCHAR(6
 
   describe('File system tests', () => {
     beforeAll(async () => {
-      if (!(await FS.getInfoAsync(FS.documentDirectory + 'SQLite')).exists) {
-        await FS.makeDirectoryAsync(FS.documentDirectory + 'SQLite');
-      }
+      await FS.deleteAsync(FS.documentDirectory + 'SQLite', { idempotent: true });
+      await FS.makeDirectoryAsync(FS.documentDirectory + 'SQLite', { intermediates: true });
     });
 
     it('should work with a downloaded .db file', async () => {
-      await FS.downloadAsync(
-        Asset.fromModule(require('../assets/asset-db.db')).uri,
-        `${FS.documentDirectory}SQLite/downloaded.db`
-      );
+      const asset = await Asset.fromModule(require('../assets/asset-db.db')).downloadAsync();
+      await FS.copyAsync({
+        from: asset.localUri,
+        to: `${FS.documentDirectory}SQLite/downloaded.db`,
+      });
```

Comment:
the previous `FS.downloadAsync()` does not work in release build with expo-updates